common/reviews/api/rush-lib.api.md

```diff
@@ -126,16 +126,21 @@ export class CredentialCache {
 }
 
 // @beta
-export type CustomTipId = 'TIP_RUSH_INCONSISTENT_VERSIONS' | string;
+export enum CustomTipIdEnum {
```

Comment:
Can you leave this as a merged type, or make this a const object?

Reply:
Yes, I could. But I am curious what's the problem with the enum? The reason why I remove the "| string" is for the IntelliSense.

Reply:
Interesting. But when you explicitly assign the value for the enum, it seems type-safe (which is what I am doing for `CustomTipIdEnum`). See this example (https://www.typescriptlang.org/play?ts=4.8.4#code/KYOwrgtgBAogHgQwgBwDbBuaBvAUFKAQSgF4oByQ8gGnygCFSL7zcBfXXAYwHsQBnAC5QAbglRhgALliIU6TJCbkAwuQDcQA) 
```ts
enum ExampleEnum {
  A = 'A',
  B = 'B'
}

const value: ExampleEnum = 'C'; // Type '"C"' is not assignable to type 'ExampleEnum'.(2322)
```

Do you think we should still make it a const object to avoid issues?

Reply:
Guess that's fine, then.

Probably should rename this

```suggestion
export enum CustomTipId {
```

Reply:
renamed